rustbridge/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "rustbridge"
 version = "0.1.0"
-authors = ["erichall <hallstrom.eric@gmail.com>"]
-
-[lib]
-name = "lib"
-path = "src/lib.rs"
+authors = ["technetos <mrgould93@gmail.com>"]
```

Comment:
any reason this just changes the author? couldn't you add yourself so there's two?

Reply:
Oops.  I generated a brand new cargo project because the original code was just hacked together.  I will add the other authors :smile: